index.html

```diff
@@ -2520,6 +2519,15 @@ <h3><dfn>Writing content</dfn></h3>
                 If |source|'s records [= list/is empty =], [= exception/throw =]
                 a {{TypeError}} and abort these steps.
               </li>
+              <li>
+                Increment |recordsDepth| by one.
+              </li>
+              <li>
+                If |recordsDepth| > `2`, [= exception/throw =] a {{TypeError}} and abort these steps.
+                <p class="issue" data-number="620">
+                  Infinite loops are not allowed by Web IDL.
```

Comment:
They will never really be infinite in practice as they have to be described by a user, using JavaScript

Reply:
Would you update this text?

Reply:
To prevent infinite loops, Web IDL bans dictionaries from including themselves as members. As the data dictionaries will have to have been created by a developer before initiating the function call, these will in practice be finite, so we work around this by using `any` as the member type and enforcing limits on allowed sub-records.

Reply:
THanks!